docs/api/apiv3/components/schemas/file_link_model.yml

```diff
@@ -45,18 +53,26 @@ properties:
         allOf:
           - $ref: './link.yml'
           - description: |-
-              This file link
+              This file link.
               
               **Resource**: FileLink
             readOnly: true
       storage:
+        allOf:
+          - $ref: './link.yml'
+          - description: |-
+              The storage resource of the linked file.
+              
+              **Resource**: Storage
+            readOnly: true
+      storageUrl:
         allOf:
           - $ref: './link.yml'
           - description: |-
               The storage url the file link references to.
               
               **Resource**: N/A
-            readOnly: true
+            writeOnly: true
       container:
         allOf:
           - $ref: './link.yml'
```

Comment:
`container` should be allowed to be written at least on the creation.

Reply:
Why is it needed? The container is defined on the route of the `POST`. So, it is not needed in the request body model.

Reply:
resolved